R/remove_funs.R

```diff
@@ -32,12 +32,12 @@ halveString <- function (string, pattern, first = TRUE, colnames=c("X1", "X2") )
   if(first)  {
     ret <- as.matrix(data.frame(X1 = unlist(lapply(allSplit, FUN = function (l) { l[1]})),
                                 X2 = unlist(lapply(1:length(allSplit), FUN = function (l) {
-                                  if(length(allSplit[[l]]) == 1 )  { ret <- NA } else { ret <- substring(string[l], first = nchar(allSplit[[l]][1])+2 ) }
+                                  if(length(allSplit[[l]]) == 1 )  { ret <- NA } else {ret <- substring(string[l], first = nchar(allSplit[[l]][1])+1+nchar(pattern) ) }
```

Comment:
not sure what happens here exactly; but seems reasonable?

Reply:
tests should be added for `halveString()` in general, but I can do this later

Reply:
If the string "a__b__c" should be splitted at first occurrence of pattern "__", the result should be c("a", "b__c"). First element of result equals first element of strsplit("a__b__c"). For the second element, however, we must use substring("a__b__c", first = x). And x equals the sum of nchar(unlist(strsplit("a__b__c", "__"))[1]) + nchar("__").